alphapept/interface.py

```diff
@@ -1102,12 +1114,12 @@ def parallel_execute(
                 memory_available = psutil.virtual_memory().available/1024**3
                 n_processes = max((int(memory_available //25 ),1))
                 logging.info(f'Using Bruker Feature Finder. Setting Process limit to {n_processes}.')
-            elif ext.lower() == '.raw':
+            elif ext.lower() in ('.raw','.mzml'):
```

Comment:
This is a very tricky fix. In general a mzml will be much bigger than a Thermo .raw. Realistically, the meemory usage will often be between that of Bruker and Thermo, although mzml can actually hold IMS data as well, making it even bigger than Bruker (funnily enough, this refers to Waters, who save their raw data in a ".raw folder" which is great confusion with .raw files and .d folders). Perhaps the safer option is to look at the actual file size instead of only at the extension?